packages/webapp/src/pages/Integration/FlowPage.tsx

```diff
@@ -89,8 +90,9 @@ export default function FlowPage(props: FlowPageProps) {
         const blob = await response.blob();
         const url = window.URL.createObjectURL(blob);
         const link = document.createElement('a');
+        const timestamp = Math.floor(new Date().getTime() / 1000).toString();
         link.href = url;
-        link.download = 'nango-integrations.zip';
+        link.download = timestamp + '-nango-integrations.zip';
```

Comment:
I would go for `nango-integration-TIMESTAMP.zip` but I am nitpicking

Reply:
Agreed!

Reply:
Hi @khaliqgant , just updated the pull request with the notes above .